src/external_app/external_messaging.ts

```diff
@@ -35,6 +35,13 @@ interface EMOutgoingMessageConfigGet extends EMMessage {
   type: "config/get";
 }
 
+interface EMOutgoingMessageScanQRCode extends EMMessage {
+  type: "qr_code/scan";
```

Comment:
Should we also tell the app what kind of QR codes we want to scan, what to accept, or does the app give the frontend everything it finds, and we tell it if that is something we accept/scan, so if we are done and it can remove the scanner?

I'm thinking about 2 QR codes side by side on a package, you dont want the app to give the wrong code to the frontend.

Reply:
I think we can start with accepting everything. We can expand it later to limit the things it accept. I would love for the frontend to get a generic QR code scanner that could work with a new concept of QR discovery for `manifest.json`.